catanatron_core/catanatron/json.py

```diff
@@ -36,6 +36,8 @@ class GameEncoder(json.JSONEncoder):
     def default(self, obj):
         if obj is None:
             return None
+        if isinstance(obj, int):
```

Comment:
some of these feel redundant - wouldn't JSON serialize these correctly? (that is, shouldn't we call `super()` if the object is not a Graph/Game or tuple?